scripts/lib/clean_venv_cache.py

```diff
@@ -1,4 +1,9 @@
 #!/usr/bin/env python3
+
+# TODO: After switching from pip to uv, we no longer create
+# /srv/zulip-venv-cache or symlink zulip-py3-venv, so this script can be
+# replaced with shutil.rmtree("/srv/zulip-venv-cache").
```

Comment:
This is probably something we could mark as a `TODO/compatibility` for when you can't directly upgrade from 9.x to `main`?

Reply:
Production upgrades aren’t relevant here. The only reason to wait is for development environment downgrades.